src/github.com/mozilla-services/pushgo/simplepush/emcee_store.go

```diff
@@ -544,7 +547,10 @@ func (s *EmceeStore) FetchAll(uaid string, since time.Time) ([]Update, []string,
 	keys := make([]string, 0, 20)
 
 	for _, chid := range chids {
-		key, _ := s.IDsToKey(uaid, chid)
+		key, err := s.IDsToKey(uaid, chid)
+		if err != nil {
```

Comment:
What does it mean if there's an error here? Is this something we should log? Does it mean our storage system might be faulty in some manner?